src/routes.tsx

```diff
@@ -170,12 +171,7 @@ export const reactPageRoutes: ReactPageRoute[] = [
   {
     path: LOCALE_PATH_PATTERN + '/new-member',
     render: ({ match }: RouteComponentProps<any>) => (
-      <EmbarkRoot
-        language={match.params.locale}
-        name={undefined}
-        baseUrl={undefined}
-        showLanding={true}
-      />
+      <Landing language={match.params.locale} />
```

Comment:
We could also remove the `language` prop here and replace it with using `useCurrentLocale` inside the `Landing` component where `language` is used now (which is only one place) -  might make this even more clean 😀

Reply:
Aha, `language` and `currentLocale` are the same things!